packages/run-protocol/src/vaultFactory/liquidateMinimum.js

```diff
@@ -55,14 +57,20 @@ const start = async zcf => {
       // check whether swapIn liquidated assets until getOfferResult() returns.
       // Of course, we also can't exit the seat until one or the other
       // liquidation takes place.
-      await deposited;
-      await E(liqSeat).getOfferResult();
+      const [offerResult, amounts, _] = await Promise.all([
```

Comment:
No use destructuring a value that won't be used.
```suggestion
      const [offerResult, amounts] = await Promise.all([
```

Reply:
My theory is that it will help prevent a future edit that adds a 4th argument from incorrectly binding the 3rd argument rather than the 4th.

Reply:
I buy that. But I expect the type checker and unit tests to catch if someone makes that erroneous edit so I personally wouldn't encumber the actual code with it. Not requesting it change.